static/index.html

```diff
@@ -487,14 +513,26 @@ <h3 class="mb-5">Other switches</h3>
               clientOptions.audience = _self.audience;
             }
 
-            if (_self.organization) {
+            if (_self.organization && !_self.useOrgAtLogin) {
               clientOptions.organization = _self.organization;
             }
 
             if (_self.useCustomStorage) {
               clientOptions.cache = sessionStorageCache;
             }
 
+            _self.clientOptions = JSON.stringify(clientOptions, null, 2);
+
+            if (_self.organization) {
+              const claims = {
+                id_token: {
+                  org_id: { values: [_self.organization] }
+                }
+              };
+
+              clientOptions.claims = JSON.stringify(claims);
+            }
```

Comment:
This is setting the `claims` parameter for the authentication call. Used when using `node-oidc-provider` as the OP, ignored by Auth0 completely (looks like it doesn't support it).